infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -23,13 +19,41 @@ Following the steps [here](https://learn.microsoft.com/en-us/cli/azure/authentic
 
 ## Terraform Backend
  
-Terraform backend state can be stored locally or in Azure blob store. To configure this you will be setting the variables like those in `infra/recipes/terraform/azure/backend.tfvars.template`. These variables are the ones found when setting up the storage account.
+Terraform backend state can be stored locally or in Azure blob store.
+
+### Storing state locally
+
+By default, the state files are simply stored locally. If this is your preffered method, no action is needed.
+
+### Storing state remotely
```

Comment:
Place this section in the footer/ end of wiki in appendix section. People follow these instructions line-by-line and might get confused. If you want you can place a pointer to that section here.

Reply:
Done